src/modules/createProject.js

```diff
@@ -0,0 +1,8 @@
+import Project from './projectClass';
+
+export default function createProject(form) {
+  const { projectName } = form.elements;
```

Comment:
Make use of ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)). I have noticed your use of the function keyword in most of your modules where you declared functions. You could change these to ES6+ arrow functions.